tests/test_create_actions_config.py

```diff
@@ -0,0 +1,38 @@
+"""This module tests the githubactions module"""
+import os
+
+from configator import create_actions_config
+
+
+def test_create_configator_file_creates_github_actions(mocker):
+    """ Testing to see if a file is created through github_actions"""
+    mock_open = mocker.mock_open()
+    #take in buildin open function and replace with mock function
+    mocker.patch('builtins.open', mock_open)
+    mocker.patch("os.makedirs")
+    create_actions_config.create_configator_file()
+    mock_open.assert_called_once_with(".github/workflows/grade.yml","w", encoding='utf-8')
```

Comment:
You mock `os.makedirs` in this test, but don't assert that the folder structure is created; this test and the one below should probably be merged, as they run the exact same code in the exact same way, and simply assert two different things.